Backend/test_project/order/models.py

```diff
@@ -19,11 +19,8 @@ class Order(models.Model):
         ('IN', 'Installments')
     ]
     order_id = models.UUIDField(primary_key= True, default=uuid.uuid4, editable=False)
-    name = models.ManyToManyField(Product, verbose_name='Назва товару', related_name='product_name')
-    id = models.ManyToManyField(Product, verbose_name='Номер товару', related_name='product_id')
-    price = models.ManyToManyField(Product, verbose_name='Ціна', related_name='product_price')
+    name = models.ManyToManyField(Product, verbose_name='Назва товару', related_name='product')
     quantity = models.IntegerField(default=1, verbose_name='Кількість')
```

Comment:
You need to replace the Name field with Products, and the related name with Orders